src/views/WorkspaceDetail.vue

```diff
@@ -87,16 +107,23 @@
           </v-card>
         </v-flex>
 
-        <v-flex md6 px-5 py-3>
-          <v-card color="transparent" flat text>
+        <v-flex
+          md6
+          px-5
+          py-3
+        >
```

Comment:
```suggestion
        <v-flex
          class="md6 px-5 py-3"
        >
```

Reply:
The `class` keyword isn't needed and reduces readability in this case. I can remove the linebreaks though.

Reply:
This is the only place that it's formatted like this though, right? Aren't all of the others as classes?

Reply:
Turns out the linter doesn't want that many attributes on one line, so I'll leave this as is.